src/cli/argv_parser.ts

```diff
@@ -120,7 +121,8 @@ const ArgvParser = {
       .option('--fail-fast', 'abort the run on first failure', false)
       .option(
         '-f, --format <TYPE[:PATH]>',
-        'specify the output format, optionally supply PATH to redirect formatter output (repeatable)',
+        'specify the output format, optionally supply PATH to redirect formatter output (repeatable).  Available formats: ' +
```

Comment:
```suggestion
        'specify the output format, optionally supply PATH to redirect formatter output (repeatable).  Available formats:\n' +
```
to make sure the first formatter is on a new line